ci/bazelisk.sh

```diff
@@ -15,12 +15,13 @@ echo "Running bazelisk in $(pwd)."
 
 # An additional bazelrc must be synthesized to specify precisely how to use the
 # GCP bazel cache.
+GCP_CREDS_FILE="$GCP_BAZEL_CACHE_KEY_SECUREFILEPATH"
```

Comment:
I am assuming that the `_SECUREFILEPATH` is added by `DownloadSecureFile` to the name? It's not really clear in the documentation https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/download-secure-file-v1?view=azure-pipelines

Reply:
`DownloadSecureFile` will store the path to the file in a variable called `$(GCP_BAZEL_CACHE_KEY.secureFilePath)`, and Azure also has a rule that you can access `$(AZURE_VARIABLES.WITH.SCOPES)` by replacing `.`s with `_`s